students/morgan/session03/mailroom2.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+# name, donation total, number of donations and average of donations
+
+infile = 'donors.txt'
+
+donors = {}
+
+with open(infile) as donor_input:
+    donor_input.readline()
+    for line in donor_input:
+        # line = line.strip()
+        name_string = line.split('-')[0].strip()
```

Comment:
you are splitting twice, so maybe:

`name_string, donation_string = line.split('-')`